src/xml_db/dump/mod.rs

```diff
@@ -163,137 +150,63 @@ impl DumpXml for Database {
     }
 }
 
-impl DumpXml for Meta {
+impl DumpXml for Times {
     fn dump_xml<E: std::io::Write>(
         &self,
         writer: &mut EventWriter<E>,
         inner_cipher: &mut dyn Cipher,
     ) -> Result<(), xml::writer::Error> {
-        writer.write(WriterEvent::start_element("Meta"))?;
-
-        SimpleTag("Generator", "keepass-rs").dump_xml(writer, inner_cipher)?;
-
-        // TODO DatabaseName
-        // TODO DatabaseNameChanged
-        // TODO DatabaseDescription
-        // TODO DatabaseDescriptionChanged
-        // TODO DefaultUserName
-        // TODO DefaultUserNameChanged
-        // TODO DeletedObjects
-        // TODO MaintenanceHistoryDays
-        // TODO Color
-        // TODO MasterKeyChanged
-        // TODO MasterKeyChangeRec
-        // TODO MasterKeyChangeForce
-        // TODO MemoryProtection
-        // TODO CustomIcons
-        // TODO RecycleBinEnabled
-        // TODO RecycleBinUUID
-        // TODO RecycleBinChanged
-        // TODO EntryTemplatesGroup
-        // TODO EntryTemplatesGroupChanged
-        // TODO LastSelectedGroup
-        // TODO LastTopVisibleGroup
-        // TODO HistoryMaxItems
-        // TODO HistoryMaxSize
-        // TODO SettingsChanged
-        // TODO CustomData
+        writer.write(WriterEvent::start_element("Times"))?;
```

Comment:
@sseemayer at some point I think we should extract the xml tag names as constants, since they are repeated in multiple places in the codebase.